build.gradle

```diff
@@ -425,6 +425,24 @@ task "bigtop-slaves"(type:Exec,
   commandLine command
 }
 
+task "docker-package"(type:Exec,
+        description: 'Build packages via docker build.\n' +
```

Comment:
You can use triple-quotes for multi-line strings instead of doing string concatenations Java-style. 
Something like
```"""Build packages via docker build.
Usage:
  $ ./gradlew""" ```and so on.

Reply:
Good suggestion! Let me refine the patch.